troposphere/static/js/models/Size.js

```diff
@@ -5,11 +5,12 @@ import moment from "moment";
 export default Backbone.Model.extend({
     urlRoot: globals.API_V2_ROOT + "/sizes",
 
-    parse: function(response) {
-        response.mem = response.mem / 1024;
-        response.start_date = moment(response.start_date);
-        response.end_date = moment(response.end_date);
-        return response;
+    parse: function (response) {
+        return Object.assign({}, response, {
```

Comment:
It's a clever way to avoid the mutation, something Connor suggested (so hat-tip 🎩) to him. 

📖 more on [`Object.assign(...)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign)